block-node/verification/src/main/java/org/hiero/block/node/verification/session/impl/DummyVerificationSession.java

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+package org.hiero.block.node.verification.session.impl;
+
+import static java.lang.System.Logger.Level.INFO;
+import static java.lang.System.Logger.Level.TRACE;
+
+import com.hedera.pbj.runtime.ParseException;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import java.util.ArrayList;
+import java.util.List;
+import org.hiero.block.internal.BlockItemUnparsed;
+import org.hiero.block.internal.BlockUnparsed;
+import org.hiero.block.node.spi.blockmessaging.BlockSource;
+import org.hiero.block.node.spi.blockmessaging.VerificationNotification;
+import org.hiero.block.node.verification.session.VerificationSession;
+
+// TODO, before going live we should remove the Dummy Implementation.
+/**
+ * Dummy implementation of VerificationSession for testing purposes.
+ * It simply collects block items and returns a successful verification notification
+ * if the last block item has a block proof.
+ */
+public class DummyVerificationSession implements VerificationSession {
```

Comment:
WHy did we create a new Session vs just simplify the Preview one like you have here?